onnxruntime/wasm/api.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
```

Comment:
Add unit tests for wasm APIs

Reply:
add tests from javascript side, will add in v1.8